### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dilipchauhan14**
Removed irrelevant lines and added greetings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,5 @@
 # HelpDesk
 
-Helps you. :))))
```
Comment (dilipchauhan14) [inline README.md]: Following Are Irrelevant lines.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,5 @@
...
-More to know.
-Good work.
-
-# webhooks payload url https://s9u0wdufd0.execute-api.us-east-2.amazonaws.com/default/github_pull_request
-
-Done,.
+Help Desk would help you.
```
Comment (dilipchauhan14) [inline README.md]: Greetings.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,5 @@
 # HelpDesk
```
Comment (dilipchauhan14) [inline README.md]: Title

File: README.md [hunk 4]
```diff
@@ -1,9 +1,5 @@
...
 
-Helps you. :))))
-More to know.
```
Comment (dilipchauhan14) [inline README.md]: Useless Lines

File: README.md [hunk 5]
```diff
@@ -1,9 +1,5 @@
...
-Good work.
-
-# webhooks payload url https://s9u0wdufd0.execute-api.us-east-2.amazonaws.com/default/github_pull_request
```
Comment (dilipchauhan14) [inline README.md]: No need

File: README.md [hunk 6]
```diff
@@ -1,9 +1,5 @@
...
-
-Done,.
+Help Desk would help you.
+Enjoy.
```
Comment (sammarth-kapse) [inline README.md]: Good Comment.

File: README.md [hunk 7]
```diff
@@ -1,9 +1,5 @@
...
+Have a Good Day.
```
Comment (sammarth-kapse) [inline README.md]: Greeting is acceptable.

File: README.md (hunk 2 above)
Comment (sammarth-kapse) [inline README.md]: Info is alright.

File: README.md (hunk 6 above)
Comment (sammarth-kapse) [inline README.md]: Remove Enjoy

**Pull request merged**